### DISCUSSION TITLE
Showtime East

### DISCUSSION THREAD

**Discussion by Loading617** [iptv-org/community#734] [Q&A]
Channel Name: Showtime
Channel ID: ShowtimeEast.us
Country: United States
Language: English
Website: sho.com
Stream URL (optional):
Notes (optional):

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/blob/aec1bdb0a7037bf6f7112d2be3bdd93a50d5209f/streams/us_moveonjoy.m3u#L68-L69
